python/lsst/__init__.py

```diff
@@ -1,3 +1,3 @@
 import pkgutil
 import lsstimport
-__path__ = pkgutil.extend_path(__path__, __name__)
+__path__ = pkgutil.extend_path(__path__, __name__)
```

Comment:
I didn't touch this file manually, so it must have been @fred3m's script. Is this intended behavior?